content/600-about/100-prisma/10-releases.mdx

```diff
@@ -76,7 +76,7 @@ Beginning with version `3.x.x`, Prisma adheres strictly to the [SemVer](https://
 
 Here is a brief overview of how Prisma's follows SemVer:
 
-- Breaking changes in stable surface (i.e. [General Availability](https://www.prisma.io/docs/about/releases#generally-available-ga)) will only be introduced in new `MAJOR` releases.
+- Breaking changes in stable surface (i.e. [General Availability](#generally-available-ga)) will only be introduced in new `MAJOR` releases.
```

Comment:
Using the external URL also hid a link that was linking to an old URL that we are now redirecting somewhere else. 
Using the internal URL the link checker found this and failed with "broken link".